app.py

```diff
@@ -88,33 +88,58 @@ def return_code(self):
         return self._return_code
 
 
-@app.route('/execute', methods=['POST'])
-def execute():
+def _execute(request_as_string: str):
     success = False
-    req = request.get_json()
     try:
         conn = get_conn_through_sentinel()
-        response = conn.execute_command(*req['command'].split())
+        response = conn.execute_command(*request_as_string.split())
         success = True
     except (redis.exceptions.ConnectionError, redis.exceptions.ResponseError):
         try:
             reload_username_password_from_file_system_if_needed(app)
             conn = get_conn_through_sentinel()
-            response = conn.execute_command(*req['command'].split())
+            response = conn.execute_command(*request_as_string.split())
             success = True
         except Exception as err:
             response = 'Exception: cannot connect. %s' % str(err)
             app.logger.exception("execute err")
     except Exception as err:
         response = 'Exception: %s' % str(err)
         app.logger.exception("execute err")
+    return response, success
+
+
+@app.route('/execute', methods=['POST'])
+def execute():
+    req = request.get_json()
+    response, success = _execute(req['command'])
 
     return jsonify({
         'response': response,
         'success': success
     })
 
 
+@app.route('/batch_execute', methods=['POST'])
+def batch_execute():
+    all_succeeded = True
+    responses = dict()
```

Comment:
Actually, a `dict` is not ideal here because there could be multiple input commands which are similar.
I'd just use a list of responses, with their index matching the index of the input command.